### ISSUE TITLE
Remote schema relations arguments type does not accept ENUM type

### ISSUE THREAD

**Issue by Ibrahim9595** [hasura/graphql-engine#6259]
I am trying to make a Remote Schema Relationships the remote Query has the following signature
`Query.ListPaymentMethods(
limit: Float
cursor: String
type: PaymentType!
): [CustomerCardType!]!`
the PaymentType is an ENUM type I want to have a remote relationship to call this mutation given a fixed type but I keep getting the following error
`in table "organizations": in remote relationship"cards": cannot validate remote relationship because expected type "PaymentType" but got "String"`
here is a screen shot
![error](https://user-images.githubusercontent.com/17384354/100340734-4ca71780-2fe4-11eb-9b9b-0f157ed502e3.png)

**Comment by sebastiancichos** [comment]
Same here bumping

**Comment by tirumaraiselvan** [comment]
You are right, this is not supported yet. Enums should be deduced from string values.

**Comment by johnshroff** [comment]
This is still not supported? :(
